src/Tasks/Common/ConflictResolution/ResolvePackageFileConflicts.cs

```diff
@@ -66,72 +67,114 @@ protected override void ExecuteCore()
                 compilePlatformItems = TargetFrameworkDirectories.SelectMany(tfd =>
                 {
                     return frameworkListReader.GetConflictItems(Path.Combine(tfd.ItemSpec, "RedistList", "FrameworkList.xml"), log);
-                });
+                }).ToList();
```

Comment:
Elsewhere we use ToArray, any reason for using List here, if so should we do the same below?